RecoMuon/DetLayers/interface/MuonDetLayerGeometry.h

```diff
@@ -83,20 +107,33 @@ class MuonDetLayerGeometry : public DetLayerGeometry{
 
   /// return all backward (-Z) layers (CSC+RPC+GEM), inside-out
   const std::vector<const DetLayer*>& allBackwardLayers() const;
```

Comment:
why is this block commented out? (it's a duplicate of the implementation above).
please remove or uncomment and rename/implement as described